### PR TITLE
Add ingress for dashboard

### PR DESCRIPTION
**Pull request by Gallardot**
### Check List 

Tests 

 - Unit test
 - E2E test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has Go code change
 - Has CI related scripts change
 - Has Terraform scripts change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: helm/chaos-mesh/README.md [hunk 1]
```diff
@@ -65,6 +65,14 @@ The following tables list the configurable parameters of the Chaos Mesh chart an
 | `dashboard.env.LISTEN_PORT` | | `2333` |
 | `dashboard.env.DATABASE_DRIVER`| The db drive used for Chaos Dashboard, support db: sqlite3, mysql| `sqlite3` |
 | `dashboard.env.DATABASE_DATASOURCE`| The db dsn used for Chaos Dashboard | `/data/core.sqlite` |
+| `dashboard.ingress.enabled`                   | Enable the use of the ingress controller to access the dashboard                         | `false`             |
+| `dashboard.ingress.certManager`               | Add annotations for cert-manager                                                      | `false`             |
```
Comment (yeya24) [inline helm/chaos-mesh/README.md:69]: Could you please update this description?

Comment (Gallardot) [reply helm/chaos-mesh/README.md:69]: How about changing this description to `Enable the cert-manager for ingress` ?

@yeya24

Comment (yeya24) [reply helm/chaos-mesh/README.md:69]: This is good. It is better to be `Enable Cert-Manager for ingress` since `the` is redundant.

Comment (Gallardot) [reply helm/chaos-mesh/README.md]: Done！

**Pull request merged**